cmd/service/execute.go

```diff
@@ -96,14 +101,28 @@ func getTaskKey(cmd *cobra.Command, serviceID string) string {
 	return taskKey
 }
 
-func getJSON(cmd *cobra.Command) string {
-	json := cmd.Flag("json").Value.String()
-	if json == "" {
-		if survey.AskOne(&survey.Input{Message: "Enter the filepath to the inputs"}, &json, nil) != nil {
-			os.Exit(0)
-		}
+func getData(cmd *cobra.Command) (string, error) {
+	data := cmd.Flag("data").Value.String()
+	jsonStr := cmd.Flag("json").Value.String()
+	jsonFile := cmd.Flag("json-file").Value.String()
+
+	if (data != "" && jsonStr != "") ||
+		(data != "" && jsonFile != "") ||
+		(jsonStr != "" && jsonFile != "") {
+		return "", errors.New("data, json and json-file flags are  mutually exclusive")
```

Comment:
Can you write something more user friendly like: "You can use only of one of the following flags: 'data', 'json' and 'json-file'."

Reply:
Sure, done.

As xpflag is eXtension to [spf13/pflag](https://github.com/spf13/pflag) that contains custom flags (implements [pflag.Value interface](https://godoc.org/github.com/spf13/pflag#Value)) and I put it in different package.